client/core/core.go

```diff
@@ -2568,7 +2568,7 @@ func (c *Core) createTokenWallet(tokenID uint32, token *asset.Token, form *Walle
 	}
 
 	return &db.Wallet{
-		Type:     "token",
+		Type:     form.Type,
```

Comment:
This makes it a bit more consistent with how other wallets are created, I think.